app/models/g5_authenticatable/user.rb

```diff
@@ -38,7 +38,7 @@ def update_roles_from_auth(auth_data)
     end
 
     def clients
-      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).resolve
+      G5Updatable::ClientPolicy::Scope.new(self, G5Updatable::Client).clients_from_client_and_location_roles
```

Comment:
Well, my thought was that they are currently two different sets. Currently resolve returns the list of clients for whom the user has direct roles (that might still be helpful). And now I'm adding a method that gives all clients for whom a client has a role on either the client or one of it's locations. Make sense.  I'm up for changing the structure... but I don't thing we want to loose the original functionality. @maeve can you comment?